src/modules/users/components/TokenActivation/StakesTab/StakesTab.css

```diff
@@ -23,8 +24,14 @@
   outline: 0;
 }
 
-.link {
-  font-size: var(--size-tiny);
+a {
+  height: 100%;
+  width: 100%;
+}
```

Comment:
This is impacting all `a href` links, causing side-effects. See examples, below:

### Current master

![2a75a805-9127-46f5-87ea-9aacd0c7dee2](https://user-images.githubusercontent.com/33682027/155968898-bc2c4ff9-4103-46dd-8c84-51321206ccc9.png)

### This branch

![ba521c7c-ce12-4023-bf22-535cc80e9d61](https://user-images.githubusercontent.com/33682027/155968908-a8ec82f2-dbab-4243-8ba4-cdfcfb3b34ec.png)